mars_craters_starting_kit.ipynb

```diff
@@ -88,8 +88,12 @@
     "\n",
     "In order to correct for the distortions and recover the circularity of the craters, the map has to be reprojected locally. For that, we use a partitions known as _quadrangles_. 140 quadrangles are used to cover the whole Mars surface and we have used only a few of them for this exercice.\n",
     "\n",
-    "<img src=\"img/quadrangles.png\" width=\"95%\">\n",
-    "<div style=\"text-align: center\">Quadrangle division in the raw THEMIS projection</div>\n",
+    "<img src=\"img/train_test_quadrangles.png\" width=\"95%\">\n",
+    "<div style=\"text-align: center\">\n",
+    "Longitude vs latitude quadrangle division in the raw THEMIS projection.<br/>\n",
+    "Challenge datasets have been constructed using random patches from these quadrangles.<br/>\n",
+    "Numbers indicate quadrangle IDs and colors refer to the train/test and public/private distinction.\n",
```

Comment:
Shouldn't we keep this in principle private? (as it reveals which data we use for testing)

Reply:
@kegl seems ok to show it.
It only reveals the location. The patches are still random.
Do you think it could have an impact ? I guess we can catch that looking at the code of the participants. But otherwise I'm ok to remove it.

Reply:
I don't think it will have an impact (the actual labels are still private, so it is not that people can fully create the testing data to train their algorithm), more out of principle (I suppose that the idea is that the new database with labels becomes public at some point?)